changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
+          "ApiChange": "Resource",
+          "ChangedApiName": "crossTenantAccessPolicy",
+          "ChangeType": "Addition",
+          "Description": "Added the `allowedCloudEndpoints` property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
+          "Target": "crossTenantAccessPolicy"
+        }
+      ],
+      "Id": "34b0a990-a3c9-4b70-92ac-a8ac93bf4860",
```

- Aligning the values with the [guidance](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog)
- Same GUID for all objects in the changelog record

```suggestion
          "ApiChange": "Property",
          "ChangedApiName": "allowedCloudEndpoints",
          "ChangeType": "Addition",
          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
          "Target": "crossTenantAccessPolicy"
        }
      ],
      "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
```